### DISCUSSION TITLE
If Alertmanager receives 'resolved' state for an alert in same group_interval, what does it do after group_interval elapses?

### DISCUSSION THREAD

**Discussion by ktnvaish22** [prometheus/alertmanager#4020] [Q&A]
Which one of these does of do?

1. Send 'firing' and' resolved'
2. Send only 'resolved'
3. Send nothing

I feel, it should send nothing, ideally.

**Comment by grobinson-grafana** [comment]
It depends on whether the last notification was firing or resolved for this alert. If firing, it sends a resolved notification. If resolved, it does nothing. If there was no previous notification for this alert, it also does nothing. I hope this helps!

**Comment by ktnvaish22** [comment]
Thanks @grobinson-grafana 
I intended to ask the third case you mentioned i.e. if a fresh notification was 'fired' in already existing group and the 'resolve' notification was also received in same group_interval.
Thanks for confirming that it won't do anything!!
